app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -618,6 +620,8 @@ class BrowserTabViewModel @Inject constructor(
                     )
                 }
             }.launchIn(viewModelScope)
+
+        browserViewState.value = currentBrowserViewState().copy(showAutofill = autofillStore.autofillAvailable)
```

Comment:
We could just add `showAutofill = autofillStore.autofillAvailable` in `initializeViewStates()` rather than updating the browserViewState twice on init{}

Reply:
That's what I almost did, but noticed none of the other properties were set that way, each truly being given a default value then later configured in init. Didn't know if that was important or not, but figured it best to match the convention.

Reply:
what do you mean? I see that for example: ```
browserViewState.value = BrowserViewState().copy(
            addToHomeVisible = addToHomeCapabilityDetector.isAddToHomeSupported(),
        )
```

Reply:
Sorry, i mis-read your suggestion as initialising in the `BrowserViewState` constructor.

I understand you now, and yeah, fully agree. Good suggestion.